src/Libraries/PythonNodeModelsWpf/IronPythonCompletionData.cs

```diff
@@ -89,31 +69,17 @@ public System.Windows.Media.ImageSource Image
         // Use this property if you want to show a fancy UIElement in the drop down list.
         public object Content
         {
-            get { return this.Text; }
+            get { return Text; }
         }
 
-        // description
-        private string _description;
-
-        public object Description
-        {
-            get
-            {
-                // lazily get the description
-                if (_description == null)
-                {
-                    _description = provider.GetDescription(this.Stub, this.Text, this.IsInstance).TrimEnd('\r', '\n');
```

Comment:
Is it okay to get rid of this logic `provider.GetDescription(this.Stub, this.Text, this.IsInstance).TrimEnd('\r', '\n')`?

Reply:
yes, it's safe, this type should have as little logic as possible. This logic already exists in the `PythonCodeCompletionDataCore.Description` property that we use to set this objects description in the constructor.

`PythonCodeCompletionDataCore` is an `IExternalCompletionData` object which the constructor on line 50 now takes, and that is the only constructor for this object.